code/modules/clothing/suits/armor.dm

```diff
@@ -94,11 +94,13 @@
 	armor = list(melee = 65, bullet = 30, laser = 50, energy = 10, bomb = 25, bio = 0, rad = 0)
 	pocket_slots = 4//More slots because coat
 
-/obj/item/clothing/suit/armor/hos/jensen
+/obj/item/clothing/suit/storage/toggle/armor/hos/jensen

```

Comment:
This path change strips the armor & body parts covered, no? Those are at `/obj/item/clothing/suit/armor/hos`.

Reply:
Agreed. I will add it.

Reply:
You'll need to set the value of this type too in `worths_list.dm`.